.github/workflows/release.yaml

```diff
@@ -48,3 +48,10 @@ jobs:
           args: release --rm-dist --timeout 1h
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Push latest Docker tag
+        run: |
+          ./scripts/push-latest.sh otel/opentelemetry-collector ${{ github.ref_name }}
```

Comment:
Isn't there a way to add this to goreleaser's config? Like here:

https://github.com/open-telemetry/opentelemetry-collector-releases/blob/1af799780977b1ef24d628ed3665a64d704f6263/.goreleaser.yaml#L235-L239

Reply:
Based on https://goreleaser.com/customization/docker/#customization I think we can just add `otel/opentelemetry-collector:latest` in the appropriate place?

Reply:
Yes, but note that the "docker" section is for the arch-specific images, whereas the "docker_manifests" is the meta image holding the arch-specific ones. The meta one is the one people pull most of the time.